src/NzbDrone.Core/Notifications/Mailgun/MailgunProxy.cs

```diff
@@ -0,0 +1,66 @@
+using System.Net;
+using NLog;
+using NzbDrone.Common.Http;
+using HttpMethod = NzbDrone.Common.Http.HttpMethod;
+
+namespace NzbDrone.Core.Notifications.Mailgun
+{
+    public interface IMailgunProxy
+    {
+        void SendNotification(string tittle, string message, MailgunSettings settings);
+    }
+
+    public class MailgunProxy : IMailgunProxy
+    {
+        private const string BaseUrlEu = "https://api.eu.mailgun.net/v3";
+        private const string BaseUrlUs = "https://api.mailgun.net/v3";
+        private readonly IHttpClient _httpClient;
+        private readonly Logger _logger;
+
+        public MailgunProxy(IHttpClient httpClient, Logger logger)
+        {
+            _httpClient = httpClient;
+            _logger = logger;
+        }
+
+        public void SendNotification(string title, string message, MailgunSettings settings)
+        {
+            try
+            {
+                var request = BuildRequest(settings, $"{settings.SenderDomain}/messages", HttpMethod.POST, title, message).Build();
+                _httpClient.Execute(request);
+            }
+            catch (HttpException ex)
+            {
+                if (ex.Response.StatusCode == HttpStatusCode.Unauthorized)
+                {
+                    throw new MailgunException("Unauthorised - ApiKey is invalid");
```

Comment:
Spelling, unauthorized

Reply:
Thanks, being from the UK I'd have never spelt unauthorised with a Z so wouldn't catch it!

Reply:
You guys and your s’s :P ... yea either is correct, it’ll just be consistent across other uses with unauthorized

Reply:
Yeah I agree consistency would be good... either that or we start an en-gb set of translation strings for the americanisms and change the us language to en-us...

Reply:
We talked about it a while back, ta didn’t think it necessary. More work for little benefit

Reply:
I would definitely agree with that to be honest, I don't mind seeing the US spellings and more products aren't bothering